### PR TITLE
fix: add user agent header

### PR DESCRIPTION
**Pull request by gris**
Should we delete the header `TursoCliVersion`?

closes https://github.com/chiselstrike/turso-cli/issues/517

### CODE REVIEW

**Comment by CodingDoug** [comment]
If we do, I think it should conform to the typical user agent format.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

> User-Agent: <product> / <product-version> <comment>

**Comment by gris** [comment]
Thanks for bringing that to my awareness. Changed it, would love to hear comments about the new format :)

File: internal/turso/turso.go [hunk 1]
```diff
@@ -76,6 +77,7 @@ func (t *Client) newRequest(method, urlPath string, body io.Reader) (*http.Reque
 		req.Header.Add("Authorization", fmt.Sprint("Bearer ", t.token))
 	}
 	req.Header.Add("TursoCliVersion", t.cliVersion)
+	req.Header.Add("User-Agent", fmt.Sprintf("Turso CLI/%s (%s/%s)", t.cliVersion[1:], runtime.GOOS, runtime.GOARCH))
```
Comment (athoscouto) [inline internal/turso/turso.go:80]: ⛏️: WDYT about using the repo name?
```suggestion
	req.Header.Add("User-Agent", fmt.Sprintf("turso-cli/%s (%s/%s)", t.cliVersion[1:], runtime.GOOS, runtime.GOARCH))
```

**Review by athoscouto — APPROVED** [review]

File: internal/turso/turso.go (hunk 1 above)
Comment (gris) [reply internal/turso/turso.go:80]: lgtm!

**Pull request merged**